src/utils/dashboard.ts

```
@@ -1,74 +1,59 @@
-import { ButtonStyle, EmbedBuilder, ActionRowBuilder, ButtonBuilder } from "discord.js"
+import { EmbedBuilder, ActionRowBuilder } from "@discordjs/builders"
+import { Emojis } from "@seedcord/gateway"
+import { ButtonStyle, ButtonBuilder } from "discord.js"
```

ESLint

[error] Import component builders from '`@discordjs/builders`'. The discord.js re-export is a separate CJS copy that breaks instanceof and toJSON when nested in a seedcord component.

(`@seedcord/no-djs-builder-import`)

---

[error] Unable to resolve path to module 'discord.js'.

(import-x/no-unresolved)